message_ix/model/MESSAGE/model_core.gms

```diff
@@ -1600,8 +1610,12 @@ ACTIVITY_CONSTRAINT_UP(node,tec,year,time)$( map_tec_time(node,tec,year,time)
 ***
 ACTIVITY_SOFT_CONSTRAINT_UP(node,tec,year,time)$( soft_activity_up(node,tec,year,time) )..
     ACT_UP(node,tec,year,time) =L=
-        SUM((vintage,mode)$( map_tec_lifetime(node,tec,vintage,year) AND map_tec_act(node,tec,year,mode,time) ),
-            ACT(node,tec,vintage,year,mode,time) ) ;
+        SUM((vintage,mode,year2)$( map_tec_lifetime(node,tec,vintage,year) AND map_tec_act(node,tec,year,mode,time)
```

Comment:
per a strange observation by @OFR-IIASA, the validation that the technology is active in a given year must match the year of operation (ie the index used in `ACT`) 

```suggestion
        SUM((vintage,mode,year2)$( map_tec_lifetime(node,tec,vintage,year2) AND map_tec_act(node,tec,year2,mode,time)
```